docs/index.rst

```diff
@@ -68,16 +69,15 @@ within the `#singer-tap-development`_ and `#singer-target-development`_ Slack ch
    stream_maps
    batch
    sinks
-   implementation/index
    testing
 
 .. toctree::
-   :caption: Development
+   :caption: SDK Development
    :maxdepth: 1
    :hidden:
 
-   CONTRIBUTING
```

Comment:
Why remove this?

Reply:
Moved under "development" parent.

<img width="240" alt="image" src="https://user-images.githubusercontent.com/18150651/217685495-59a04d29-1965-477e-a7fd-426bd5aed76a.png">

If we raise up, we should raise up "Release Process" also; otherwise the "Development" parent would only have a single parent.

Reply:
Optionally, we could just have "Contributor's Guide", "Release Process", and "Deprecation Timeline" as top-level items under the "SDK Development" section.

@edgarrmondragon - any objections to us moving them up to the top level?

Reply:
As noted in the description, I do think "Deprecation Timeline" should be visible to non-contributors, since that effects tap/target maintainers even if they are not themselves contributors.

Reply:
Oh I see where it is now. I was just failing to see the new location.

I have a slight preference for moving it to the top-level under "SDK Development". There's only 2 items nested there after all, and the name they're nested under is identical to the category name above.

Reply:
> any objections to us moving these up to the top level?

@aaronsteers no objections, it makes sense :+1: